cdc/generation.cc

```diff
@@ -26,9 +26,9 @@
 
 #include "keys.hh"
 #include "schema_builder.hh"
```

Comment:
Why should cdc stop using partitioners?

Reply:
Because it just needs sharding which does not depend on partitioner any more. I will add the explanation to the commit message.